db/migrate/20181019185328_create_projects.rb

```diff
@@ -0,0 +1,11 @@
+class CreateProjects < ActiveRecord::Migration[5.2]
+  def change
+    create_table :projects do |t|
+      t.string :name, null: false, default: ''
+      t.string :url, null: false, default: ''
+      t.datetime :project_created_at
+      t.datetime :project_last_modified
+      t.timestamps null: false
```

Comment:
Rails handles this field itself, do you really need to avoid this to be null?

Reply:
Good point @cone I'm gonna remove it